network/definition/data/components/vcrear/vcrear-message.yaml

```
@@ -76,3 +76,17 @@ messages:
       shockpotdispRL:
       shockpotVoltRR:
       shockpotdispRR:
+
+  wheelSpeed:
+    description: Rear wheel speeds
+    id: 0x460
+    cycleTimeMs: 50
+    signals:
+      wheelVelocityL:
+        template: wheelLinearVelocity
+      wheelVelocityR:
+        template: wheelLinearVelocity
+      wheelRpmL:
+        template: wheelRPM
+      wheelRpmR:
+        template: wheelRPM
```

rpm is a unit and probably shouldn't be in the signal name (unless you have two signals that are otherwise identical other than the unit, which is not the case here). Also, I know that vcfront publishes fronts and vcrear publishes rears, but it might be nice to have F/R in here regardless. I think it would be more obvious if you used
```suggestion
      wheelSpeedRL:
        template: wheelRPM
      wheelSpeedRR:
        template: wheelRPM
```
and same for fronts